### PR TITLE
refactor: Upgrade json-logic app service to use V3

### PR DESCRIPTION
**Pull request by marcpfuller**
**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-examples/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [x] I have fully tested (add details below) this the new feature or bug fix (if not, why?)

## Testing Instructions
Start edgex with no-secty
run `./app-service  -cp -o -dev`
verify service starts up

### CODE REVIEW

**Review by jim-wang-intel — APPROVED** [review]

File: application-services/custom/json-logic/res/configuration.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+Writable:
+  LogLevel: 'INFO'
+  Writable.Telemetry:
+    Interval: "30s"
+    PublishTopicPrefix : "edgex/telemetry" # /<service-name>/<metric-name> will be added to this Publish Topic prefix
```
Comment (lenny-intel) [inline application-services/custom/json-logic/res/configuration.yaml:5]: Remove, no longer exists
```suggestion
```

File: application-services/custom/json-logic/res/configuration.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+
+Service:
+  Host: 'localhost'
+  Port: 59780 # Adjust if running multiple examples at the same time to avoid duplicate port conflicts
+  StartupMsg: 'This is a sample application service using the JSON logic transform'
+
```
Comment (lenny-intel) [inline application-services/custom/json-logic/res/configuration.yaml:12]: Missing MessageBus section for client ID override

```
MessageBus:
  Disabled: false  # Set to true if not using edgex-messagebus Trigger below and Metrics are not needed
  Optional:
    ClientId: "app-json-logic"
```

File: application-services/custom/json-logic/res/configuration.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    Trigger.EdgexMessageBus.SubscribeHost:
+      Host: "localhost"
+      Port: 6379
+      Protocol: "redis"
+      SubscribeTopics: "edgex/events/#"
+    Trigger.EdgexMessageBus.PublishHost:
+      Host: "localhost"
+      Port: 6379
+      Protocol: "redis"
+      PublishTopic: "example"
```
Comment (lenny-intel) [inline application-services/custom/json-logic/res/configuration.yaml:31]: Replace with similar to this: https://github.com/edgexfoundry/app-service-configurable/blob/main/res/sample/configuration.yaml#L152-L155

Comment (marcpfuller) [reply application-services/custom/json-logic/res/configuration.yaml]: done

File: application-services/custom/json-logic/res/configuration.yaml (hunk 2 above)
Comment (marcpfuller) [reply application-services/custom/json-logic/res/configuration.yaml:11]: done

**Review by jim-wang-intel — APPROVED** [review]

**Review by lenny-intel — APPROVED** [review]
LGTM

**Pull request merged**